plugin/pkg/scheduler/algorithm/BUILD

```diff
@@ -24,6 +24,7 @@ go_library(
         "//plugin/pkg/scheduler/schedulercache:go_default_library",
         "//vendor:k8s.io/apimachinery/pkg/apis/meta/v1",
         "//vendor:k8s.io/apimachinery/pkg/labels",
+        "//vendor:k8s.io/client-go/pkg/apis/apps",
```

Comment:
does this help you https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/apps/v1beta1/types.go#L37-L50 ?